exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,15 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
```

Comment:
Hi Azan excellent job! This works, but small suggestion you could simplify the code. I did same as you but changed my code after I got feedback. If the mark is greater or equal to 80, then you code will exit at line 13. So you don't need to do the check 'mark < 80'on line 14. it could be like : 
function calculateGrade(mark) {
if (mark >= 80) {
return "A";
} else if (mark >= 60) {
return "B"
} else if (mark >= 50) {
return "C";
} else {
return "F";
}}

Reply:
Great eye Mya, thanks for the feedback I will change it as soon as I can.